### ISSUE TITLE
Get the directory structure compatible with Three.js structure...

### ISSUE THREAD

**Issue by trusktr** [lume/glas#70]
...eventually, and make the workflow almost the same, with the same docs/examples site, so that we can render everything in a way that's familiar to the existing community?

**Comment by trusktr** [comment]
This will be a big undertaking (to port all the examples to work with the AS version)

**Comment by ZoeDreams** [comment]
I concur. A lot of the examples three uses have been around longer then three's implementation. I feel that keeping the API and nomenclature the same is important for clarity between three and glas users.

**Comment by trusktr** [comment]
Yeah. Let's do this once dev has settled a little. Then we'll do it in one step before doing moving onto new things.
